frontend/.env.local.example

```
@@ -1,14 +1,17 @@
 NEXT_PUBLIC_API_URL=http://localhost:8080
 NEXT_PUBLIC_APP_URL=http://localhost:3000
 
+# Database (shared Postgres with the Spring backend; used by better-auth + drizzle-kit)
+DATABASE_URL=postgresql://user:password@localhost:5432/disscount
+
 # Better Auth
 BETTER_AUTH_SECRET=your_better_auth_secret_here
 BETTER_AUTH_URL=your_app_url_here # for example: http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)